tests/Doctrine/Tests/Models/CMS/CmsUser.php

```diff
@@ -155,6 +155,10 @@ class CmsUser
      */
     public $email;
     /**
+     * @OneToOne(targetEntity="CmsAvatar", mappedBy="user", cascade={"persist"}, orphanRemoval=true)
+     */
+    public $avatar;
```

Comment:
Sure- Is there a good candidate? Something with a relation and some numeric scalar on a related entity is what I needed.